### ISSUE TITLE
database sql file not found

### ISSUE THREAD

**Issue by lagkdok** [schongabor/php-cms#1]
hi,
i've just download the script but i didn't find any sql file for the database to try it.
please, commit

**Comment by schongabor** [comment]
hi,

you have to set up your own database connection and change the connection params in includes/db.php

**Comment by lagkdok** [comment]
yes, I know this but the sql file is missing, so there is no database for the script to connect with

**Comment by schongabor** [comment]
Now i got it. I'll upload it asap!
